global.json

```diff
@@ -619,6 +619,7 @@
       "priority": 0.30,
       "name": "WFM Demo Project",
       "description": "A demo project showcasing the reusable WFM modules. Note: Using openshift.feedhenry.com access via your OpenShift account is limited to using the small OpenShift Gear. Due to WFM template's non-trivial functionality you may have stability issues using the small gear.",
+      "docs": "https://github.com/feedhenry-staff/wfm-doc/blob/master/Getting-Started.md",
```

Comment:
@JameelB @vchepeli yes, it should link to the getting started guide directly as this is the relevant one for users importing the project template.